drivers/hmis/app/graphql/types/hmis_schema/has_enrollments.rb

```diff
@@ -36,9 +37,10 @@ def resolve_enrollments(scope = object.enrollments, **args)
 
       private
 
-      def scoped_enrollments(scope, sort_order: :most_recent, include_in_progress: false, open_on_date: nil, project_types: nil, search_term: nil)
+      def scoped_enrollments(scope, sort_order: :most_recent, exclude_wip: false, wip_only: false, open_on_date: nil, project_types: nil, search_term: nil)
```

Comment:
I think you have an odd edge case where you can say "exclude WIP" AND "only WIP." That would return nothing.  Maybe that should take an enum instead of a boolean?
`[ALL_ENROLLMENTS, WIP_ONLY, NON_WIP_ONLY]`  Seems like there should be a case statement in there.

Reply:
Alternatively, you could just preference one, so if `exclude_wip` was true, you could ignore `wip_only` but that's not as clear.

Reply:
Yeah that's a fair point. I'll update it to use an enum.